integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesEvents.java

```diff
@@ -741,7 +741,7 @@ void testK8SEventsStartStopWatchingNS(boolean enableClusterRoleBinding) {
     upgradeAndVerifyOperator(opNamespace, opParams);
 
     logger.info("verify NamespaceWatchingStopped event is logged in namespace {0}", domainNamespace2);
-    checkNamespaceWatchingStoppedEvent(opNamespace, domainNamespace2, null, "Normal", timestamp,
+    checkNamespaceWatchingStoppedEvent(opNamespace, domainNamespace2, domainUid, "Normal", timestamp,
```

Comment:
domainUid should be null as before. NamespaceWatchingStopped event will not contin domainUID